### ISSUE TITLE
[Feature Request] Shortcut for adding Labels

### ISSUE THREAD

**Issue by dmlls** [alainm23/planify#589]
**Is your feature request related to a problem? Please describe.**
Adding a label to a task is currently quite cumbersome, especially when adding a new task, having first to write the new task, then press Enter, then click on the task, click on the Labels icon, and finally select the label.

**Describe the solution you'd like**
Add a shortcut (e.g., Ctrl+L) to label a task quickly at the time of writing it.
In summary, how I think it could work:
- Press `a` to add a new Task.
- Write the task.
- Press `Ctrl+L` to open Labels menu.
- Search the label, by typing its name.
- Press `Enter` to select the Label searched.
- Press `Enter` again to add the Task.

Just want to add: Thank you very much Alain for your amazing work!

**Comment by omega3** [comment]
Would be great to have a plugin for user defined shortcuts for each label.

**Comment by P4Cu** [comment]
There's also another way which we could do it.
> Some task for #work

It could pop up dropdown with defined labels and would need to be definitely documented somewhere in the UI.

**Comment by rscircus** [comment]
I think this means the `Quick Add` feature specifically, right?

**Comment by rscircus** [comment]
I think it also ain't possible to assign a task to a project in `Quick Add`, right?

**Comment by dmlls** [comment]
Actually, I was not referring to Quick Add... This shortcut would work when adding a task within Planner itself.
